packages/extension/src/ui/features/accountTokens/useTokenBalanceForAccount.ts

```diff
@@ -25,7 +24,7 @@ interface UseTokenBalanceForAccountArgs {
  * Get the individual token balance for the account, using Multicall if available
  * This will automatically mutate when the number of pending transactions decreases
  */
-
+// Look at this
```

Comment:
Do we still need to change this `useTokenBalanceForAccount`?

Reply:
I will remove the comment :+1: